### ISSUE TITLE
Clarification 

### ISSUE THREAD

**Issue by midhun1993** [salaros/wp-purge-pagespeed-button#1]
Hi, 

This is not an issue,  i have a couple of doubts please clarify if you are available. is this a fully functioning plugin ? i reviewed your code and i don't find any code for clear the pagespeed cache (if i am wrong please correct me) and how can write a core php code for clear pagespeed service cache ?

thanks
midhun

**Comment by salaros** [comment]
Hi @midhun1993 

It's a fully functional plugin. It doesn't do any cache purging via PHP, because in case of PageSpeed module JavaScript is more suitable.

As you might know from the [official docs](https://www.modpagespeed.com/doc/system#purge_cache) the easiest way to purge the cache is to call the URL with [HTTP PURGE method](https://stackoverflow.com/questions/25857508/what-is-the-http-method-purge):


Apache:

    ModPagespeedEnableCachePurge on
    ModPagespeedPurgeMethod PURGE     (optional)

Nginx:

    pagespeed EnableCachePurge on;
    pagespeed PurgeMethod PURGE;     (optional)

So basically my plugin does what this cURL command does below:

    $ curl -X PURGE https://my-cool-website.com/*
    Purge successful

Except it uses a jQuery.ajax instead of cURL :+1:

**Comment by salaros** [comment]
You must be new to GitHub, because you haven't read the README, it clearly says not to use GutHub issues for support:

![screenshot_20171111-034817 01](https://user-images.githubusercontent.com/675348/32684369-ef1597ca-c693-11e7-9966-1ee02de0daa9.png)

Additionally you assumed my plugin is a fake, please have some respect.
